docs/runtime/minimal-runtime.md

```diff
@@ -254,21 +264,154 @@ level=info module=net-runner caller=root.go:152 ts=2021-06-14T08:42:47.219513806
 ```
 <!-- markdownlint-enable line-length -->
 
-:::info
+:::tip
 
 The local network runner will take control of the current terminal until you
 terminate it via Ctrl+C. For the rest of the guide keep the local network
 running and use a separate terminal to run the client.
 
 :::
 
+## Testing From Oasis CLI
+
+After you have the runtime running in your local network, the next step is to
+test that it actually works. Let's see, if the local network was correctly
+initialized and the runtime is ready:
+
+```bash
+${OASIS_CORE_PATH}/oasis-node control status -a unix:/tmp/minimal-runtime-test/net-runner/network/client-0/internal.sock
+```
+
+If everything is working correctly, you should see the `"status": "ready"`
+under the runtime's `"committee"` field after a while and an increasing
+`"latest_round"` value:
+
+```
+      "committee": {
+        "status": "ready",
+        "active_version": {
+          "major": 1
+        },
+        "latest_round": 19,
+        "latest_height": 302,
+        "executor_roles": null,
+```
+
+Now, let's connect Oasis CLI with our new runtime and try to transfer some
+tokens. First we'll need to find the value of `"chain_context"` field in the
+`oasis-node control status` report above (e.g.
+`8b1ddecd5e3cf0925afd5e334a6427c1676cc34ad7c2bde5bf3d27316e3189cf`). This value
+is different each time the `oasis-net-runner` is executed.
+
+Now we add a new `localhost` network to the Oasis CLI along with the chain
+domain separation context and a path to the local socket:
+
+```bash
+oasis network add localhost 8b1ddecd5e3cf0925afd5e334a6427c1676cc34ad7c2bde5bf3d27316e3189cf unix:/tmp/eth-runtime-test/net-runner/network/client-0/internal.sock
+? Description: localhost
+? Denomination symbol: ROSE
+? Denomination decimal places: 9
+```
+
+Next we add `minimal` runtime to the wallet. By default, `oasis-net-runner`
+assigns ID `8000000000000000000000000000000000000000000000000000000000000000`
+to the first provided runtime.
+
+```bash
+oasis paratime add localhost emerald 8000000000000000000000000000000000000000000000000000000000000000
+? Description: minimal
+? Denomination symbol: ROSE
+? Denomination decimal places: 9
+```
+
+Finally, we import _Alice's_ account to our wallet with a private key below:
```

Comment:
Should we update the CLI to include a nice way of using the test keys without the need for manual import? They could be referenced via a special `test:alice` or something.